arma3/@task_force_radio/addons/task_force_radio_items/stringtable.xml

```diff
@@ -1,687 +1,891 @@
-<?xml version="1.0" encoding="UTF-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This stringtable is not used anymore. Undo all your changes on this file.

Reply:
Ok, I will fix.